build_tools/benchmarks/benchmark_helper_test.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import io
+import json
+import unittest
+import benchmark_helper
+
+
+class BenchmarkHelperTest(unittest.TestCase):
+
+  def test_merge_results_simple(self):
```

Comment:
Leave some comments in `benchmark_helper.py`, I think we can probably split the test into two parts:

1. Test loading JSON string into dict (and verification)
2. Test merging a list of dict

This might reduce the StringIO usages in the tests

Reply:
Yes, thank you for suggesting that. I also considered that at some point, but I guess I was too lazy to do it. Thank you for pushing me over the edge. :grinning: